<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParagraphIndentSpacing</class>
 <widget class="QWidget" name="ParagraphIndentSpacing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>412</height>
   </rect>
  </property>
  <property name="whatsThis">
   <string>&lt;p&gt;&lt;ul&gt;&lt;li&gt;&lt;b&gt;Single&lt;/b&gt;: The normal linespacing&lt;/li&gt;&lt;/ul&gt;&lt;/p&gt; </string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>Left Indent:</string>
       </property>
       <property name="buddy">
        <cstring>left</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KoUnitDoubleSpinBox" name="left"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>Right Indent:</string>
       </property>
       <property name="buddy">
        <cstring>right</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KoUnitDoubleSpinBox" name="right"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>First Line Indent:</string>
       </property>
       <property name="buddy">
        <cstring>first</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="2" column="1">
        <widget class="KoUnitDoubleSpinBox" name="first"/>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="autoTextIndent">
         <property name="text">
          <string>Auto-infer from text size</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>Line Spacing:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="lineSpacing"/>
       </item>
       <item>
        <widget class="QStackedWidget" name="spacingStack">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="KoUnitDoubleSpinBox" name="unitsPage" />
         <widget class="QSpinBox" name="percentPage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="minimum">
           <number>85</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="useFont">
       <property name="text">
        <string>Use Font Metrics</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>Paragraph Space Before:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="KoUnitDoubleSpinBox" name="before"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel">
       <property name="text">
        <string>Paragraph Space After:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="KoUnitDoubleSpinBox" name="after"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KoUnitDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>KoUnitDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
